ext/XS-APItest/t/boolean-thr.t

```diff
@@ -0,0 +1,37 @@
+#!./perl
+
+use strict;
+use warnings;
+
+use Test::More;
+use XS::APItest;
+
+use Config ();
+BEGIN { plan skip_all => "perl does not support threads" unless $Config::Config{usethreads} };
```

Comment:
Could also be written `use if !$Config::Config{usethreads}, 'Test::More', skip_all => ...;` (done before `use Test::More;`)

Reply:
If the current CI run tests OK, I'll rewrite it as that and squash it down